### PR TITLE
Improve console commands

### PR DESCRIPTION
**Pull request by iamteapot422**
Introduce a couple of methods to remove code repetition and significantly shorten all the commands.
Also fix a little bug, where long help command was executing twice

### CODE REVIEW

**Review by stilnat — CHANGES_REQUESTED** [review]
quick review : 
hit body part seems to work okay, I tested it by cutting the right arm and then calling
`hitbodypart HumanHandRightItem(Clone) all toxic 1000`

it destroys it instantly as expected

same test for destroy body part, I get an exception

using command `destroybodypart HumanHandRightItem(Clone)`

```
NullReferenceException: Object reference not set to an instance of an object
SS3D.Systems.IngameConsoleSystem.Commands.Command.ReceiveCheckResponse[T] (System.String[] args, SS3D.Systems.IngameConsoleSystem.Commands.Command+CheckArgsResponse& response, T& calculatedValues) (at Assets/Scripts/SS3D/Systems/IngameConsoleSystem/Commands/Command.cs:67)
SS3D.Systems.IngameConsoleSystem.Commands.DestroyBodyPartCommand.Perform (System.String[] args, FishNet.Connection.NetworkConnection conn) (at Assets/Scripts/SS3D/Systems/IngameConsoleSystem/Commands/DestroyBodyPartCommand.cs:26)
SS3D.Systems.IngameConsoleSystem.CommandsController.RpcLogic___CmdProcessOnlineCommand_161581254 (System.String commandName, System.String[] args, FishNet.Connection.NetworkConnection conn) (at Assets/Scripts/SS3D/Systems/IngameConsoleSystem/CommandsController.cs:127)
```

File: Assets/Scripts/SS3D/Systems/IngameConsoleSystem/Commands/HurtCommand.cs [hunk 1]
```diff
@@ -13,50 +13,27 @@ namespace SS3D.Systems.IngameConsoleSystem.Commands
 {
     public class HurtCommand : Command
     {
-        public override string LongDescription => "Hurt a given body part and body layer of a player by a given amount of damages from a given type";
+        public override string LongDescription => "Hurt body layer in a body part of a player";
+        public override string Usage => "(game object name) (BodyLayerType) (DamageType) (amount of damage)";
```
Comment (stilnat) [inline Assets/Scripts/SS3D/Systems/IngameConsoleSystem/Commands/HurtCommand.cs]: change usage to (Player ckey)(Body part name) (body layer name) (damage type) (damage amount)
you can also add an exemple like hurt EditorUser HumanHead all toxic 10

File: Assets/Scripts/SS3D/Core/IsExternalInit.cs
Comment (stilnat) [inline Assets/Scripts/SS3D/Core/IsExternalInit.cs:1]: Where is that used ?

**Review by stilnat — CHANGES_REQUESTED** [review]
Tested about half commands on host and 5 or 6 on clients, everything works, code makes sense and reduce a lot the clutter. Please get your branch up to date and it's ready to merge.

File: Assets/Scripts/SS3D/Core/IsExternalInit.cs
Comment (iamteapot422) [reply Assets/Scripts/SS3D/Core/IsExternalInit.cs:1]: Everywhere with records and inits. Some sort of unity clutch

**Review by stilnat — APPROVED** [review]
all good, nice stuff, good work

**Pull request merged**